src/Executor/Executor.php

```diff
@@ -42,45 +49,45 @@ public function __construct(string $endpoint)
      * @param string $deploymentId
      * @param string $projectId
      * @param string $source
-     * @param string $runtime
-     * @param string $baseImage
+     * @param string $image
      * @param bool $remove
      * @param string $entrypoint
      * @param string $workdir
-     * @param string $destinaction
-     * @param string $network
-     * @param array $vars
+     * @param string $destination
+     * @param array $variables
      * @param array $commands
      */
     public function createRuntime(
         string $deploymentId,
         string $projectId,
         string $source,
-        string $runtime,
-        string $baseImage,
+        string $image,
         bool $remove = false,
         string $entrypoint = '',
         string $workdir = '',
         string $destination = '',
-        array $vars = [],
+        array $variables = [],
         array $commands = []
     ) {
+        $runtimeId = "$projectId-$deploymentId";
         $route = "/runtimes";
         $headers = [
             'content-type' => 'application/json',
-            'x-appwrite-executor-key' => App::getEnv('_APP_EXECUTOR_SECRET', '')
+            'authorization' => 'Bearer ' . App::getEnv('_APP_EXECUTOR_SECRET', ''),
+            'x-opr-runtime-id' => $runtimeId
```

Comment:
Lets define `$headers` as a class variable with `content-type` and `authorization` both set since they dont change across requests.

Then you can merge the main headers with endpoint specific headers like we do in our php sdk 
https://github.com/appwrite/sdk-for-php/blob/master/src/Appwrite/Client.php#L165

Reply:
Updated. I fill class-variable in constructor since App::get() must be done in constructor.

Please revisit and resolve if all is fine.